### PR TITLE
Added new file

### PR DESCRIPTION
**Pull request by venkat0606**

### CODE REVIEW

File: code1.py [hunk 1]
```diff
@@ -0,0 +1 @@
+print('hello all data scientists')
```
Comment (suhas2092) [inline code1.py:1]: Hey Apoorv

Comment (sammy1987) [inline code1.py:1]: he dude..wassup

Comment (ParthaT123) [inline code1.py:1]: Approve

Comment (bharath-ts) [inline code1.py:1]: Hello DS!!!!

**Pull request merged**